### PR TITLE
The great name update of this demo

### PR DESCRIPTION
**Pull request by cristeahub**
After "far and long and longer than far" as we would say in Norwegian we've landed on our name for this demo!

This prq adds the new name to the readmes and nfos!

### CODE REVIEW

File: nin.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
     "sigveseb",
     "stiaje"
   ],
-  "description": "Glorious Ninjadev Revision Submission",
-  "previewImage": "https://raw.githubusercontent.com/ninjadev/nin/master/nin/frontend/app/images/nin-dark.png",
+  "description": "Ninjadev entry for Revision 2017",
+  "previewImage": "https://github.com/ninjadev/re/blob/master/whatareyousyncingabout.png",
```
Comment (iver56) [inline nin.json]: page not found. funker det etter merge mon tro?

Comment (cristeahub) [inline nin.json]: Jepp!

Comment (iver56) [inline nin.json]: If we need a direct link to the image, https://raw.githubusercontent.com/ninjadev/re/master/whatareyousyncingabout.png is better

Comment (iver56) [inline nin.json]: https://github.com/ninjadev/re/pull/232

**Pull request merged**